src/lib/components/layout/Sidebar/UserMenu.svelte

```
@@ -4,12 +4,14 @@
 
 	import { flyAndScale } from '$lib/utils/transitions';
 	import { goto } from '$app/navigation';
-	import ArchiveBox from '$lib/components/icons/ArchiveBox.svelte';
+	import { page } from '$app/stores';
```

ESLint

[error] 'page' is defined but never used.

(@typescript-eslint/no-unused-vars)